addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1169,13 +1169,16 @@ test("html composer: basic rendering", async () => {
     await openFormView("res.partner", serverState.partnerId);
     await click("button", { text: "Send message" });
     await contains("textarea.o-mail-Composer-input[placeholder='Send a message to followers…']");
-    await contains(".o-mail-Composer-html.odoo-editor-editable", { count: 0 });
-    composerService.setHtmlComposer();
-    await contains(".o-mail-Composer-html.odoo-editor-editable");
-    await contains("textarea.o-mail-Composer-input[placeholder='Send a message to followers…']", {
+    await insertText(".o-mail-Composer-input", "Hello World!");
+    await contains(".o-mail-Composer-html.odoo-editor-editable", {
+        text: "Hello World!",
```

Comment:
This assert is making the step too specific. Here we just want to ensure there is no o-mail-Composer-html regardless of its content.